utils/install-common.sh

```diff
@@ -18,9 +18,14 @@ install_apisix_dependencies_rpm() {
 
 install_dependencies_rpm() {
     # install basic dependencies
-    yum -y install wget tar gcc automake autoconf libtool make curl git which unzip sudo
-    yum -y install epel-release
-    yum install -y yum-utils readline-devel
+    if [[ $IMAGE_BASE == "registry.access.redhat.com/ubi8/ubi" ]]; then
+        yum -y install --disablerepo=* --enablerepo=ubi-8-appstream-rpms --enablerepo=ubi-8-baseos-rpms wget tar gcc automake autoconf libtool make curl git which unzip sudo
+        yum install -y --disablerepo=* --enablerepo=ubi-8-appstream-rpms --enablerepo=ubi-8-baseos-rpms yum-utils
+    else 
+        yum -y install wget tar gcc automake autoconf libtool make curl git which unzip sudo
+        yum -y install epel-release
+        yum install -y yum-utils readline-devel
```

Comment:
Plz use `yum install -y` to install the deps.

Reply:
Functionally the flag can be put anywhere. Okay I will put the flag after `install`. Thank you.

Reply:
done